### ISSUE TITLE
Question: can exif data automatically be read and displayed?

### ISSUE THREAD

**Issue by Diederka** [coneda/kor#190]
am I rigth that Kor (or respectively image magick) automatically reads the file types and displays them 
eg: File type of the original: image/jpeg

ist it possible to read the exif data and display it ? 
like fo instance "image width" and "height" and "resolution in dpi"? 
such technical metadata is very important, and actually given but 
 some collegues didnt't collect them while scanning thus we can't display them ...

**Comment by moritzschepp** [comment]
Yes, the mime type is determined by KOR. Its actually the browser that sends the mime/type. KOR just ensures that its correct (for security).

KOR includes a module that can read the EXIF data from JPG and TIF files.

As a **feature**, I could imagine that the EXIF data is read after an image (mime type image/jpeg or image/tif) is uploaded and then written to fields for the entity type "medium" (if the fields exist).

If you want to have this quickly, have a look at the library https://github.com/remvee/exifr which is included in the KOR bundle. You can use this in a script to regularly add the information for already uploaded images.

**Comment by Diederka** [comment]
oh, a feature like this in future times would be great! In the meantimeI will have a look at the library. Thank you!

**Comment by wuebbena** [comment]
cf. https://bugs.coneda.net/issues/455
